.github/workflows/build-rdo-package.yml

```diff
@@ -26,7 +29,7 @@ jobs:
 
       - uses: actions/setup-python@v5
         with:
-          python-version: "3.10"
+          python-version: ${{ matrix.python_version }}
           cache: "pip"
```

Comment:
Because of this change/step setting the right version, I believe you could probably replace all _invocations_ of `python${{ matrix.python_version }}` with just `python`. But I'll leave it up to you :+1:

Reply:
Probably.... but it worked / packages are build with this (in another orga-repo for testing)